src/modules/attachHandler.ts

```diff
@@ -89,6 +90,7 @@ export const postHandler = async (req: NextApiRequest, res: NextApiResponse) =>
                         sectionId,
                         interviewId,
                         commentId,
+                        cvForInterviewId,
```

Comment:
but interviewId is already passed to this function

Reply:
same as in previous comment